config/server/setup.ts

```diff
@@ -0,0 +1,11 @@
+export function checkForAllEnvironmentVariables() {
+    let requiredEnvironmentVariables = [
+        'WARZONE_JWT_SECRET'
```

Comment:
Added a check here that throws an error if you have any missing, required environment variables